target/linux/ipq806x/files-5.4/arch/arm/boot/dts/qcom-ipq8062.dtsi

```diff
@@ -0,0 +1,101 @@
+#include "qcom-ipq8064.dtsi"
```

Comment:
Please add an SPDX license identifier.

Reply:
Added.

Reply:
but in the wrong commit.

Reply:
Sorry for my mistake. It should be fixed now.